aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -24,7 +24,7 @@
 #include <c10/core/SymInt.h>
 #include <c10/util/string_view.h>
 
-#if USE_ROCM
+#if defined(USE_ROCM) && (defined(USE_MEM_EFF_ATTENTION) || defined(USE_FLASH_ATTENTION))
```

Comment:
@AngryLoki can you explain why USE_ROCM must be replaced with defined(USE_ROCM)?

Reply:
Style/consistency reasons, now there are only 3 places with `#if USE_ROCM` and 509 places with `defined(USE_ROCM)` (+263 `#ifdef USE_ROCM`, +64 `#ifndef USE_ROCM`) in the whole codebase.

Reply:
This looks good @AngryLoki :) I think the reason why it make sense to keep it as if USE_ROCM is that if you have defined USE_ROCM=0 the defined(USE_ROCM) will evaluate to true as far as I know, maybe someone can correct me.